### DISCUSSION TITLE
Can Speechly detect different people talking using the same Microphone input?

### DISCUSSION THREAD

**Discussion by IanChok** [speechly/speechly#212] [Q&A] [answered]
Hello,

I was wondering given a single microphone in a physical conference room, can Speechly differentiate between different people speaking?

**Comment by aukkonen** [comment, marked as answer]
Hello @IanChok!

Thank you for posting the question. At the moment we do not do speaker diarization, unfortunately.

Would this be a crucial feature for your use-case?

**Reply by cfry** [reply]
I could make use of that functionality.
Here's what would make sense to me:
On the dashboard for my app, under training data.
there would be a section for "training speakers".
TO create a new speaker, you'd enter a string Identifier, ie "John".
then click a button to start listening.
The John reads some text that you display.
After that, when you get back a segment in your JS application,
the segment would have a field  person: "John"
or perhaps
person: "unknown"
maybe a "confidence percent" would be returned as well,
or perhaps even a confidence percent for all the speakers defined in the app.
